generic/build.vars

```diff
@@ -8,7 +8,7 @@ TAP_WINDOWS_VERSION="${TAP_WINDOWS_VERSION:-9.21.2}"
 OPENVPN_VERSION="${OPENVPN_VERSION:-2.3.11}"
 OPENVPN_GUI_VERSION="${OPENVPN_GUI_VERSION:-10}"
 
-OPENSSL_URL="${OPENSSL_URL:-http://www.openssl.org/source/openssl-${OPENSSL_VERSION}.tar.gz}"
+OPENSSL_URL="${OPENSSL_URL:-https://www.openssl.org/source/openssl-${OPENSSL_VERSION}.tar.gz}"
```

Comment:
This looks reasonable, but I suggest renaming the commit message and making it fit into about 72 characters. Right now it overflows and will make thinks like "git shortlog" look silly.

Maybe just change the commit message to "Fetch OpenSSL sources via https"?